### PR TITLE
Make code working with new OpenAPI packaging

### PR DESCRIPTION
**Pull request by nfx**
Changes the OpenAPI packages

### CODE REVIEW

**Comment by pietern** [comment]
@nfx Is this ready to go or a WIP?

**Comment by nfx** [comment]
@pietern yep, done

File: internal/mlflow_test.go [hunk 1]
```diff
@@ -58,103 +69,130 @@ func TestAccMLflowRuns(t *testing.T) {
...
-	require.NoError(t, err)
-
-	all, err := w.RegisteredModels.ListAll(ctx, mlflow.ListRegisteredModelsRequest{})
+	all, err := w.ModelRegistry.ListModelsAll(ctx, ml.ListModelsRequest{})
 	require.NoError(t, err)
 	assert.True(t, len(all) >= 1)
 }
 
+func deleteModel(t *testing.T, w *databricks.WorkspaceClient, ctx context.Context, created *ml.CreateModelResponse) {
+	if deleteWorkaroundEnabled {
```
Comment (pietern) [inline internal/mlflow_test.go:123]: What's this? Should prob track it in an issue.

**Review by pietern — APPROVED** [review]

File: internal/mlflow_test.go (hunk 1 above)
Comment (nfx) [reply internal/mlflow_test.go:123]: that's what we've been discussing on standup today :) we'll remove it, once MLflow service is compliant

**Pull request merged**